x-pack/plugins/security_solution/server/lib/timeline/saved_object_mappings/migrations/notes.test.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { TIMELINE_ID_REF_NAME } from '../../constants';
+import { migrateNoteTimelineIdToReferences, TimelineId } from './notes';
+
+describe('notes migrations', () => {
+  describe('7.16.0 timelineId', () => {
+    it('removes the timelineId from the migrated document', () => {
+      const migratedDoc = migrateNoteTimelineIdToReferences({
+        id: '1',
+        type: 'awesome',
+        attributes: { timelineId: '123' },
+      });
+
+      expect(migratedDoc.attributes).toEqual({});
+      expect(migratedDoc.references).toEqual([
+        // importing the timeline saved object type from the timeline saved object causes a circular import and causes the jest tests to fail
+        { id: '123', name: TIMELINE_ID_REF_NAME, type: 'siem-ui-timeline' },
```

Comment:
Is `siem-ui-timeline` stored in a constant anywhere? I can see us changing all of these to security-* later on

Reply:
Yeah there's a constant but when I import it, it caused a circular import or something because the jest test just started failing 🤷‍♂️

Reply:
great, lol. Thanks. Will take a look at it later